src/main/java/org/opensearch/sdk/rest/BaseExtensionRestHandler.java

```diff
@@ -36,6 +36,10 @@
  * Provides convenience methods to reduce boilerplate code in an {@link ExtensionRestHandler} implementation.
  */
 public abstract class BaseExtensionRestHandler implements ExtensionRestHandler {
+    /**
+     * Constant for JSON content type
+     */
+    public static final String JSON_CONTENT_TYPE = "application/json";
```

Comment:
Please use the built-in constant from Apache HTTP core.
```
import static org.apache.hc.core5.http.ContentType.APPLICATION_JSON;

public static final String JSON_CONTENT_TYPE = APPLICATION_JSON.toString();
```

Reply:
Or, `APPLICATION_JSON.withCharset(StandardCharsets.UTF_8).toString()` ... I like this one. 

```java
System.out.println(org.apache.hc.core5.http.ContentType.APPLICATION_JSON.withCharset(StandardCharsets.UTF_8).toString());
```
> application/json; charset=UTF-8